tests/FakeItEasy.Analyzer.Tests.Shared/FakeItEasy.Analyzer.Tests.Shared.projitems

```diff
@@ -9,8 +9,6 @@
     <Import_RootNamespace>FakeItEasy.Analyzer.Tests</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(MSBuildThisFileDirectory)Helpers\CodeFixVerifier.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Helpers\CodeFixVerifier.Helper.cs" />

```

Comment:
They're unused now because the analyzer doesn't provide code fixes, but if we want to add code fixes, we'll be glad to have those files... Of course, we can always find them in the git history.